programs/programs/ma/cmsp/calculator.py

```
@@ -12,8 +12,10 @@ class MaCmsp(ProgramCalculator):
     - https://www.mass.gov/childrens-medical-security-plan
     """
 
-    # $239/month matches MaMassHealth child coverage values (YOUNG_CHILD, OLDER_CHILD, INFANT)
-    member_amount = 239
+    # $239/month × 12 = $2,868/year per eligible uninsured child.
```

Ruff (0.15.2)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)